src/composites/FeaturesContainer/Overview/OverviewCollector.ts

```diff
@@ -0,0 +1,103 @@
+import { overviewData } from './index.ts';
+import { State } from "../../../state.ts";
+import { useAtomValue } from "jotai";
+
+//expected DSStats for compact-output: 0-0-0-0-6 / 1-1-0-0-34 / 4-0-1-0-161
+
+export function OverviewDataCollector():overviewData {
+    const dataset = useAtomValue(State.dataset);
+
+    const overviewData: overviewData = {
+        qualityAspectsCount: [0, 0, 0, 0, 0],
+        qualityFactorsCount: [0, 0, 0, 0, 0],
+        measuresCount: [0, 0, 0, 0, 0],
+        qualityAspectNodes: [],
+        qualityFactorNodes: [],
+        measureNodes: [],
+    };
+
+    if(dataset) {
```

Comment:
can we add some comments to explain what the code is trying to achieve here?

Reply:
Comments added throughout